TENDERMINT_INTEGRATION.rst

```diff
@@ -9,7 +9,7 @@ Running a single node with ``docker-compose``
 
 .. code-block:: bash
 
-    $ docker-compose -f docker-compose.tendermint.yml up bdb
+    $ docker-compose up -d bdb
```

Comment:
do we still need this file? I thought documentation was going to be part of `contributing.md` cc @ttmc

Reply:
I'm not sure what the file `TENDERMINT_INTEGRATION.rst` was for because I gave up trying to understand all the Docker Compose stuff some time ago.

But yes, let's try to have all the documentation for "how to run a BigchainDB node" in the new "Contributing to BigchainDB" docs created by pull request #2119 . That would include all instructions for how to run a BigchainDB node using Docker Compose and the new set of Docker Compose files.

No more special "floater" files. Just one set of docs about all-ways to Contribute to BigchainDB, including how to Develop BigchainDB, and everyone doing that needs to run a local node somehow... hence the need for some docs about how to run a local node. (There is more than one way.)

Reply:
makes sense, I will add it to the relevant docs once your PR #2119 is merged. Meanwhile, I will keep this a dependency.

Reply:
#2119 is now merged

Reply:
Will be addressed in the next commit.

Reply:
remove this file and add relevant docs to `CONTRIBUTING.md`